hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/BasicAuthProviderTest.java

```diff
@@ -0,0 +1,36 @@
+package pl.allegro.tech.hermes.consumers.consumer.sender.http;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+
+import org.apache.commons.codec.binary.Base64;
+import org.junit.Test;
+
+import pl.allegro.tech.hermes.api.AuthenticationType;
+import pl.allegro.tech.hermes.api.DeliveryType;
+import pl.allegro.tech.hermes.api.EndpointAddress;
+import pl.allegro.tech.hermes.api.Subscription;
+import pl.allegro.tech.hermes.consumers.consumer.Message;
+import pl.allegro.tech.hermes.consumers.test.MessageBuilder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class BasicAuthProviderTest {
```

Comment:
why did you rewrite it to java? we try to write new stuff in groovy (per contributors guide)